src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationserver/data/model/Client.kt

```diff
@@ -50,12 +54,21 @@ data class Client(
   @Column(length = 2000)
   @Convert(converter = TokenSettingsConverter::class)
   var tokenSettings: TokenSettings,
+
+  @OneToMany
+  @JoinColumn(name = "registeredClientId")
+  @Where(clause = "access_token_issued_at =(select max(oa.access_token_issued_at) from oauth2_authorization oa where oa.authorization_grant_type='client_credentials')")
+  val latestClientCredentialsAuthorization: MutableSet<Authorization> ? = mutableSetOf(),
```

Comment:
Does this field need to be optional if we're initialising it to an empty set? You would need to confirm how it behaves when there are no authorizations present to be sure

Reply:
Removed default value